Makefile

```diff
@@ -38,6 +39,16 @@ lint:
 	find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" | xargs gofmt -d -s
 	go mod verify
 
+########################################
+### Protobuf
+
+proto-gen: proto-update-deps
+	@echo "Generating Protobuf files"
+	$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace tendermintdev/sdk-proto-gen sh ./scripts/protocgen.sh
+
+proto-update-deps:
+	GO111MODULE=off go get github.com/stormcat24/protodep
+	protodep up
```

Comment:
For managing 3rd-party proto files (depedencies), I'm trying to use [protodep](https://github.com/stormcat24/protodep). It's my first time to try it, but it's simple to use. Please see details in the `protodep.toml` file below.